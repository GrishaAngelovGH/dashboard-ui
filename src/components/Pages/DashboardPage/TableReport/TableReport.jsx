import { Table } from 'antd'

export const columns = [
	{
		title: 'Name',
		dataIndex: 'name'
	},
	{
		title: 'Activations',
		dataIndex: 'activations',
		sorter: {
			compare: (a, b) => a.activations - b.activations,
			multiple: 3
		}
	},
	{
		title: 'Orders',
		dataIndex: 'orders',
		sorter: {
			compare: (a, b) => a.orders - b.orders,
			multiple: 2
		}
	},
	{
		title: 'Active Users',
		dataIndex: 'users',
		sorter: {
			compare: (a, b) => a.users - b.users,
			multiple: 1
		}
	}
]

export const data = [
	{
		key: '1',
		name: 'Product 1',
		activations: 10,
		orders: 50,
		users: 90
	},
	{
		key: '2',
		name: 'Product 2',
		activations: 20,
		orders: 60,
		users: 100
	},
	{
		key: '3',
		name: 'Product 3',
		activations: 30,
		orders: 70,
		users: 110
	},
	{
		key: '4',
		name: 'Product 3',
		activations: 40,
		orders: 80,
		users: 120
	}
]

const TableReport = () => (
	<Table bordered={true} columns={columns} dataSource={data} />
)

export default TableReport