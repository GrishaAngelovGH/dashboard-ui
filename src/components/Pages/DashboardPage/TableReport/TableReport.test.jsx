import { render } from '@testing-library/react'

import TableReport from './TableReport'

window.matchMedia = window.matchMedia || function () {
    return {
        matches: false,
        addListener: function () { },
        removeListener: function () { }
    }
}

test('should render TableReport component', () => {
    const view = render(<TableReport />)

    expect(view).toMatchSnapshot()
})