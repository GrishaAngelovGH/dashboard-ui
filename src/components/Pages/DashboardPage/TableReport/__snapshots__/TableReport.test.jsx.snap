// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`should render TableReport component 1`] = `
{
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="ant-table-wrapper css-dev-only-do-not-override-19lec04"
      >
        <div
          class="ant-spin-nested-loading css-dev-only-do-not-override-19lec04"
        >
          <div
            class="ant-spin-container"
          >
            <div
              class="ant-table ant-table-bordered css-dev-only-do-not-override-19lec04"
            >
              <div
                class="ant-table-container"
              >
                <div
                  class="ant-table-content"
                >
                  <table
                    style="table-layout: auto;"
                  >
                    <colgroup />
                    <thead
                      class="ant-table-thead"
                    >
                      <tr>
                        <th
                          class="ant-table-cell"
                          scope="col"
                        >
                          Name
                        </th>
                        <th
                          aria-label="Activations"
                          class="ant-table-cell ant-table-column-has-sorters"
                          scope="col"
                          tabindex="0"
                        >
                          <div
                            aria-describedby="test-id"
                            class="ant-table-column-sorters"
                          >
                            <span
                              class="ant-table-column-title"
                            >
                              Activations
                            </span>
                            <span
                              class="ant-table-column-sorter ant-table-column-sorter-full"
                            >
                              <span
                                aria-hidden="true"
                                class="ant-table-column-sorter-inner"
                              >
                                <span
                                  aria-label="caret-up"
                                  class="anticon anticon-caret-up ant-table-column-sorter-up"
                                  role="img"
                                >
                                  <svg
                                    aria-hidden="true"
                                    data-icon="caret-up"
                                    fill="currentColor"
                                    focusable="false"
                                    height="1em"
                                    viewBox="0 0 1024 1024"
                                    width="1em"
                                  >
                                    <path
                                      d="M858.9 689L530.5 308.2c-9.4-10.9-27.5-10.9-37 0L165.1 689c-12.2 14.2-1.2 35 18.5 35h656.8c19.7 0 30.7-20.8 18.5-35z"
                                    />
                                  </svg>
                                </span>
                                <span
                                  aria-label="caret-down"
                                  class="anticon anticon-caret-down ant-table-column-sorter-down"
                                  role="img"
                                >
                                  <svg
                                    aria-hidden="true"
                                    data-icon="caret-down"
                                    fill="currentColor"
                                    focusable="false"
                                    height="1em"
                                    viewBox="0 0 1024 1024"
                                    width="1em"
                                  >
                                    <path
                                      d="M840.4 300H183.6c-19.7 0-30.7 20.8-18.5 35l328.4 380.8c9.4 10.9 27.5 10.9 37 0L858.9 335c12.2-14.2 1.2-35-18.5-35z"
                                    />
                                  </svg>
                                </span>
                              </span>
                            </span>
                          </div>
                        </th>
                        <th
                          aria-label="Orders"
                          class="ant-table-cell ant-table-column-has-sorters"
                          scope="col"
                          tabindex="0"
                        >
                          <div
                            aria-describedby="test-id"
                            class="ant-table-column-sorters"
                          >
                            <span
                              class="ant-table-column-title"
                            >
                              Orders
                            </span>
                            <span
                              class="ant-table-column-sorter ant-table-column-sorter-full"
                            >
                              <span
                                aria-hidden="true"
                                class="ant-table-column-sorter-inner"
                              >
                                <span
                                  aria-label="caret-up"
                                  class="anticon anticon-caret-up ant-table-column-sorter-up"
                                  role="img"
                                >
                                  <svg
                                    aria-hidden="true"
                                    data-icon="caret-up"
                                    fill="currentColor"
                                    focusable="false"
                                    height="1em"
                                    viewBox="0 0 1024 1024"
                                    width="1em"
                                  >
                                    <path
                                      d="M858.9 689L530.5 308.2c-9.4-10.9-27.5-10.9-37 0L165.1 689c-12.2 14.2-1.2 35 18.5 35h656.8c19.7 0 30.7-20.8 18.5-35z"
                                    />
                                  </svg>
                                </span>
                                <span
                                  aria-label="caret-down"
                                  class="anticon anticon-caret-down ant-table-column-sorter-down"
                                  role="img"
                                >
                                  <svg
                                    aria-hidden="true"
                                    data-icon="caret-down"
                                    fill="currentColor"
                                    focusable="false"
                                    height="1em"
                                    viewBox="0 0 1024 1024"
                                    width="1em"
                                  >
                                    <path
                                      d="M840.4 300H183.6c-19.7 0-30.7 20.8-18.5 35l328.4 380.8c9.4 10.9 27.5 10.9 37 0L858.9 335c12.2-14.2 1.2-35-18.5-35z"
                                    />
                                  </svg>
                                </span>
                              </span>
                            </span>
                          </div>
                        </th>
                        <th
                          aria-label="Active Users"
                          class="ant-table-cell ant-table-column-has-sorters"
                          scope="col"
                          tabindex="0"
                        >
                          <div
                            aria-describedby="test-id"
                            class="ant-table-column-sorters"
                          >
                            <span
                              class="ant-table-column-title"
                            >
                              Active Users
                            </span>
                            <span
                              class="ant-table-column-sorter ant-table-column-sorter-full"
                            >
                              <span
                                aria-hidden="true"
                                class="ant-table-column-sorter-inner"
                              >
                                <span
                                  aria-label="caret-up"
                                  class="anticon anticon-caret-up ant-table-column-sorter-up"
                                  role="img"
                                >
                                  <svg
                                    aria-hidden="true"
                                    data-icon="caret-up"
                                    fill="currentColor"
                                    focusable="false"
                                    height="1em"
                                    viewBox="0 0 1024 1024"
                                    width="1em"
                                  >
                                    <path
                                      d="M858.9 689L530.5 308.2c-9.4-10.9-27.5-10.9-37 0L165.1 689c-12.2 14.2-1.2 35 18.5 35h656.8c19.7 0 30.7-20.8 18.5-35z"
                                    />
                                  </svg>
                                </span>
                                <span
                                  aria-label="caret-down"
                                  class="anticon anticon-caret-down ant-table-column-sorter-down"
                                  role="img"
                                >
                                  <svg
                                    aria-hidden="true"
                                    data-icon="caret-down"
                                    fill="currentColor"
                                    focusable="false"
                                    height="1em"
                                    viewBox="0 0 1024 1024"
                                    width="1em"
                                  >
                                    <path
                                      d="M840.4 300H183.6c-19.7 0-30.7 20.8-18.5 35l328.4 380.8c9.4 10.9 27.5 10.9 37 0L858.9 335c12.2-14.2 1.2-35-18.5-35z"
                                    />
                                  </svg>
                                </span>
                              </span>
                            </span>
                          </div>
                        </th>
                      </tr>
                    </thead>
                    <tbody
                      class="ant-table-tbody"
                    >
                      <tr
                        class="ant-table-row ant-table-row-level-0"
                        data-row-key="1"
                      >
                        <td
                          class="ant-table-cell"
                        >
                          Product 1
                        </td>
                        <td
                          class="ant-table-cell"
                        >
                          10
                        </td>
                        <td
                          class="ant-table-cell"
                        >
                          50
                        </td>
                        <td
                          class="ant-table-cell"
                        >
                          90
                        </td>
                      </tr>
                      <tr
                        class="ant-table-row ant-table-row-level-0"
                        data-row-key="2"
                      >
                        <td
                          class="ant-table-cell"
                        >
                          Product 2
                        </td>
                        <td
                          class="ant-table-cell"
                        >
                          20
                        </td>
                        <td
                          class="ant-table-cell"
                        >
                          60
                        </td>
                        <td
                          class="ant-table-cell"
                        >
                          100
                        </td>
                      </tr>
                      <tr
                        class="ant-table-row ant-table-row-level-0"
                        data-row-key="3"
                      >
                        <td
                          class="ant-table-cell"
                        >
                          Product 3
                        </td>
                        <td
                          class="ant-table-cell"
                        >
                          30
                        </td>
                        <td
                          class="ant-table-cell"
                        >
                          70
                        </td>
                        <td
                          class="ant-table-cell"
                        >
                          110
                        </td>
                      </tr>
                      <tr
                        class="ant-table-row ant-table-row-level-0"
                        data-row-key="4"
                      >
                        <td
                          class="ant-table-cell"
                        >
                          Product 3
                        </td>
                        <td
                          class="ant-table-cell"
                        >
                          40
                        </td>
                        <td
                          class="ant-table-cell"
                        >
                          80
                        </td>
                        <td
                          class="ant-table-cell"
                        >
                          120
                        </td>
                      </tr>
                    </tbody>
                  </table>
                </div>
              </div>
            </div>
            <ul
              class="ant-pagination ant-table-pagination ant-table-pagination-right css-dev-only-do-not-override-19lec04"
            >
              <li
                aria-disabled="true"
                class="ant-pagination-prev ant-pagination-disabled"
                title="Previous Page"
              >
                <button
                  class="ant-pagination-item-link"
                  disabled=""
                  tabindex="-1"
                  type="button"
                >
                  <span
                    aria-label="left"
                    class="anticon anticon-left"
                    role="img"
                  >
                    <svg
                      aria-hidden="true"
                      data-icon="left"
                      fill="currentColor"
                      focusable="false"
                      height="1em"
                      viewBox="64 64 896 896"
                      width="1em"
                    >
                      <path
                        d="M724 218.3V141c0-6.7-7.7-10.4-12.9-6.3L260.3 486.8a31.86 31.86 0 000 50.3l450.8 352.1c5.3 4.1 12.9.4 12.9-6.3v-77.3c0-4.9-2.3-9.6-6.1-12.6l-360-281 360-281.1c3.8-3 6.1-7.7 6.1-12.6z"
                      />
                    </svg>
                  </span>
                </button>
              </li>
              <li
                class="ant-pagination-item ant-pagination-item-1 ant-pagination-item-active"
                tabindex="0"
                title="1"
              >
                <a
                  rel="nofollow"
                >
                  1
                </a>
              </li>
              <li
                aria-disabled="true"
                class="ant-pagination-next ant-pagination-disabled"
                title="Next Page"
              >
                <button
                  class="ant-pagination-item-link"
                  disabled=""
                  tabindex="-1"
                  type="button"
                >
                  <span
                    aria-label="right"
                    class="anticon anticon-right"
                    role="img"
                  >
                    <svg
                      aria-hidden="true"
                      data-icon="right"
                      fill="currentColor"
                      focusable="false"
                      height="1em"
                      viewBox="64 64 896 896"
                      width="1em"
                    >
                      <path
                        d="M765.7 486.8L314.9 134.7A7.97 7.97 0 00302 141v77.3c0 4.9 2.3 9.6 6.1 12.6l360 281.1-360 281.1c-3.9 3-6.1 7.7-6.1 12.6V883c0 6.7 7.7 10.4 12.9 6.3l450.8-352.1a31.96 31.96 0 000-50.4z"
                      />
                    </svg>
                  </span>
                </button>
              </li>
            </ul>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="ant-table-wrapper css-dev-only-do-not-override-19lec04"
    >
      <div
        class="ant-spin-nested-loading css-dev-only-do-not-override-19lec04"
      >
        <div
          class="ant-spin-container"
        >
          <div
            class="ant-table ant-table-bordered css-dev-only-do-not-override-19lec04"
          >
            <div
              class="ant-table-container"
            >
              <div
                class="ant-table-content"
              >
                <table
                  style="table-layout: auto;"
                >
                  <colgroup />
                  <thead
                    class="ant-table-thead"
                  >
                    <tr>
                      <th
                        class="ant-table-cell"
                        scope="col"
                      >
                        Name
                      </th>
                      <th
                        aria-label="Activations"
                        class="ant-table-cell ant-table-column-has-sorters"
                        scope="col"
                        tabindex="0"
                      >
                        <div
                          aria-describedby="test-id"
                          class="ant-table-column-sorters"
                        >
                          <span
                            class="ant-table-column-title"
                          >
                            Activations
                          </span>
                          <span
                            class="ant-table-column-sorter ant-table-column-sorter-full"
                          >
                            <span
                              aria-hidden="true"
                              class="ant-table-column-sorter-inner"
                            >
                              <span
                                aria-label="caret-up"
                                class="anticon anticon-caret-up ant-table-column-sorter-up"
                                role="img"
                              >
                                <svg
                                  aria-hidden="true"
                                  data-icon="caret-up"
                                  fill="currentColor"
                                  focusable="false"
                                  height="1em"
                                  viewBox="0 0 1024 1024"
                                  width="1em"
                                >
                                  <path
                                    d="M858.9 689L530.5 308.2c-9.4-10.9-27.5-10.9-37 0L165.1 689c-12.2 14.2-1.2 35 18.5 35h656.8c19.7 0 30.7-20.8 18.5-35z"
                                  />
                                </svg>
                              </span>
                              <span
                                aria-label="caret-down"
                                class="anticon anticon-caret-down ant-table-column-sorter-down"
                                role="img"
                              >
                                <svg
                                  aria-hidden="true"
                                  data-icon="caret-down"
                                  fill="currentColor"
                                  focusable="false"
                                  height="1em"
                                  viewBox="0 0 1024 1024"
                                  width="1em"
                                >
                                  <path
                                    d="M840.4 300H183.6c-19.7 0-30.7 20.8-18.5 35l328.4 380.8c9.4 10.9 27.5 10.9 37 0L858.9 335c12.2-14.2 1.2-35-18.5-35z"
                                  />
                                </svg>
                              </span>
                            </span>
                          </span>
                        </div>
                      </th>
                      <th
                        aria-label="Orders"
                        class="ant-table-cell ant-table-column-has-sorters"
                        scope="col"
                        tabindex="0"
                      >
                        <div
                          aria-describedby="test-id"
                          class="ant-table-column-sorters"
                        >
                          <span
                            class="ant-table-column-title"
                          >
                            Orders
                          </span>
                          <span
                            class="ant-table-column-sorter ant-table-column-sorter-full"
                          >
                            <span
                              aria-hidden="true"
                              class="ant-table-column-sorter-inner"
                            >
                              <span
                                aria-label="caret-up"
                                class="anticon anticon-caret-up ant-table-column-sorter-up"
                                role="img"
                              >
                                <svg
                                  aria-hidden="true"
                                  data-icon="caret-up"
                                  fill="currentColor"
                                  focusable="false"
                                  height="1em"
                                  viewBox="0 0 1024 1024"
                                  width="1em"
                                >
                                  <path
                                    d="M858.9 689L530.5 308.2c-9.4-10.9-27.5-10.9-37 0L165.1 689c-12.2 14.2-1.2 35 18.5 35h656.8c19.7 0 30.7-20.8 18.5-35z"
                                  />
                                </svg>
                              </span>
                              <span
                                aria-label="caret-down"
                                class="anticon anticon-caret-down ant-table-column-sorter-down"
                                role="img"
                              >
                                <svg
                                  aria-hidden="true"
                                  data-icon="caret-down"
                                  fill="currentColor"
                                  focusable="false"
                                  height="1em"
                                  viewBox="0 0 1024 1024"
                                  width="1em"
                                >
                                  <path
                                    d="M840.4 300H183.6c-19.7 0-30.7 20.8-18.5 35l328.4 380.8c9.4 10.9 27.5 10.9 37 0L858.9 335c12.2-14.2 1.2-35-18.5-35z"
                                  />
                                </svg>
                              </span>
                            </span>
                          </span>
                        </div>
                      </th>
                      <th
                        aria-label="Active Users"
                        class="ant-table-cell ant-table-column-has-sorters"
                        scope="col"
                        tabindex="0"
                      >
                        <div
                          aria-describedby="test-id"
                          class="ant-table-column-sorters"
                        >
                          <span
                            class="ant-table-column-title"
                          >
                            Active Users
                          </span>
                          <span
                            class="ant-table-column-sorter ant-table-column-sorter-full"
                          >
                            <span
                              aria-hidden="true"
                              class="ant-table-column-sorter-inner"
                            >
                              <span
                                aria-label="caret-up"
                                class="anticon anticon-caret-up ant-table-column-sorter-up"
                                role="img"
                              >
                                <svg
                                  aria-hidden="true"
                                  data-icon="caret-up"
                                  fill="currentColor"
                                  focusable="false"
                                  height="1em"
                                  viewBox="0 0 1024 1024"
                                  width="1em"
                                >
                                  <path
                                    d="M858.9 689L530.5 308.2c-9.4-10.9-27.5-10.9-37 0L165.1 689c-12.2 14.2-1.2 35 18.5 35h656.8c19.7 0 30.7-20.8 18.5-35z"
                                  />
                                </svg>
                              </span>
                              <span
                                aria-label="caret-down"
                                class="anticon anticon-caret-down ant-table-column-sorter-down"
                                role="img"
                              >
                                <svg
                                  aria-hidden="true"
                                  data-icon="caret-down"
                                  fill="currentColor"
                                  focusable="false"
                                  height="1em"
                                  viewBox="0 0 1024 1024"
                                  width="1em"
                                >
                                  <path
                                    d="M840.4 300H183.6c-19.7 0-30.7 20.8-18.5 35l328.4 380.8c9.4 10.9 27.5 10.9 37 0L858.9 335c12.2-14.2 1.2-35-18.5-35z"
                                  />
                                </svg>
                              </span>
                            </span>
                          </span>
                        </div>
                      </th>
                    </tr>
                  </thead>
                  <tbody
                    class="ant-table-tbody"
                  >
                    <tr
                      class="ant-table-row ant-table-row-level-0"
                      data-row-key="1"
                    >
                      <td
                        class="ant-table-cell"
                      >
                        Product 1
                      </td>
                      <td
                        class="ant-table-cell"
                      >
                        10
                      </td>
                      <td
                        class="ant-table-cell"
                      >
                        50
                      </td>
                      <td
                        class="ant-table-cell"
                      >
                        90
                      </td>
                    </tr>
                    <tr
                      class="ant-table-row ant-table-row-level-0"
                      data-row-key="2"
                    >
                      <td
                        class="ant-table-cell"
                      >
                        Product 2
                      </td>
                      <td
                        class="ant-table-cell"
                      >
                        20
                      </td>
                      <td
                        class="ant-table-cell"
                      >
                        60
                      </td>
                      <td
                        class="ant-table-cell"
                      >
                        100
                      </td>
                    </tr>
                    <tr
                      class="ant-table-row ant-table-row-level-0"
                      data-row-key="3"
                    >
                      <td
                        class="ant-table-cell"
                      >
                        Product 3
                      </td>
                      <td
                        class="ant-table-cell"
                      >
                        30
                      </td>
                      <td
                        class="ant-table-cell"
                      >
                        70
                      </td>
                      <td
                        class="ant-table-cell"
                      >
                        110
                      </td>
                    </tr>
                    <tr
                      class="ant-table-row ant-table-row-level-0"
                      data-row-key="4"
                    >
                      <td
                        class="ant-table-cell"
                      >
                        Product 3
                      </td>
                      <td
                        class="ant-table-cell"
                      >
                        40
                      </td>
                      <td
                        class="ant-table-cell"
                      >
                        80
                      </td>
                      <td
                        class="ant-table-cell"
                      >
                        120
                      </td>
                    </tr>
                  </tbody>
                </table>
              </div>
            </div>
          </div>
          <ul
            class="ant-pagination ant-table-pagination ant-table-pagination-right css-dev-only-do-not-override-19lec04"
          >
            <li
              aria-disabled="true"
              class="ant-pagination-prev ant-pagination-disabled"
              title="Previous Page"
            >
              <button
                class="ant-pagination-item-link"
                disabled=""
                tabindex="-1"
                type="button"
              >
                <span
                  aria-label="left"
                  class="anticon anticon-left"
                  role="img"
                >
                  <svg
                    aria-hidden="true"
                    data-icon="left"
                    fill="currentColor"
                    focusable="false"
                    height="1em"
                    viewBox="64 64 896 896"
                    width="1em"
                  >
                    <path
                      d="M724 218.3V141c0-6.7-7.7-10.4-12.9-6.3L260.3 486.8a31.86 31.86 0 000 50.3l450.8 352.1c5.3 4.1 12.9.4 12.9-6.3v-77.3c0-4.9-2.3-9.6-6.1-12.6l-360-281 360-281.1c3.8-3 6.1-7.7 6.1-12.6z"
                    />
                  </svg>
                </span>
              </button>
            </li>
            <li
              class="ant-pagination-item ant-pagination-item-1 ant-pagination-item-active"
              tabindex="0"
              title="1"
            >
              <a
                rel="nofollow"
              >
                1
              </a>
            </li>
            <li
              aria-disabled="true"
              class="ant-pagination-next ant-pagination-disabled"
              title="Next Page"
            >
              <button
                class="ant-pagination-item-link"
                disabled=""
                tabindex="-1"
                type="button"
              >
                <span
                  aria-label="right"
                  class="anticon anticon-right"
                  role="img"
                >
                  <svg
                    aria-hidden="true"
                    data-icon="right"
                    fill="currentColor"
                    focusable="false"
                    height="1em"
                    viewBox="64 64 896 896"
                    width="1em"
                  >
                    <path
                      d="M765.7 486.8L314.9 134.7A7.97 7.97 0 00302 141v77.3c0 4.9 2.3 9.6 6.1 12.6l360 281.1-360 281.1c-3.9 3-6.1 7.7-6.1 12.6V883c0 6.7 7.7 10.4 12.9 6.3l450.8-352.1a31.96 31.96 0 000-50.4z"
                    />
                  </svg>
                </span>
              </button>
            </li>
          </ul>
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
